<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operaciones Aritm√©ticas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(232, 255, 206);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_fondo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>380</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>250</height>
     </size>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivo/recursos.qrc">:/Logos/skt.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_pregunta">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_respuesta">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Nirmala UI&quot;;
background-color: rgb(255, 140, 25);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_enviar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 65, 255);
font: 75 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Enviar Respuesta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivo/recursos.qrc">:/Logos/ultraTlogo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivo/recursos.qrc">:/Logos/uatlogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Archivo/recursos.qrc">:/Logos/ingenierialogo.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_fondo</zorder>
   <zorder>input_respuesta</zorder>
   <zorder>boton_enviar</zorder>
   <zorder>label_pregunta</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../Archivo/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
